{
  statement.executed=false;
  try {
    context.getTransactionManager().rollback(savepoint);
  }
 catch (  Throwable e) {
    ResultSetViewer.log.debug("Error during transaction rollback",e);
  }
}
